### PR TITLE
[improve] support i18n for tab name & add copy button for backup path

### PR DESCRIPTION
**Pull request by zhoulii**
## What changes were proposed in this pull request

1. add i18n support for tab name

2. add copy button for backup path

3. change `Option Log` to `Operation Log`

## Verifying this change

- *Manually verified the change by testing locally.* 

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / **no**)

### CODE REVIEW

**Comment by zhoulii** [comment]
cc @wangsizhu0504

File: streampark-console/streampark-console-webapp/src/locales/lang/zh-CN/flink/app.ts [hunk 1]
```diff
@@ -86,6 +86,14 @@ export default {
     nullAccessToken: '访问令牌为空，请联系管理员添加.',
     invalidAccessToken: '访问令牌无效，请联系管理员。',
     detailTab: {
+      detailTabName: {
+        option: '选项',
+        configuration: '配置',
+        flinkSql: 'Flink SQL',
+        savepoint: '保存点',
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/locales/lang/zh-CN/flink/app.ts]: Whether savepoint does not require translation should be a proper term. It is best to discuss this with back-end personnel

**Review by wangsizhu0504 — APPROVED** [review]
LGTM!

File: streampark-console/streampark-console-webapp/src/locales/lang/zh-CN/flink/app.ts (hunk 1 above)
Comment (zhoulii) [reply streampark-console/streampark-console-webapp/src/locales/lang/zh-CN/flink/app.ts]: many chinese blogs use ‘保存点’ for savepoint， use '检查点' for checkpoint，but Savepoint is indeed a professional term, maybe use ‘Savepoint’ directly is a better choice.

Comment (wolfboys) [reply streampark-console/streampark-console-webapp/src/locales/lang/zh-CN/flink/app.ts]: > Whether savepoint does not require translation should be a proper term. It is best to discuss this with back-end personnel

"savepoint" is not recommended to be translated into Chinese

**Pull request merged**